server/service/src/sync/synchroniser.rs

```diff
@@ -112,6 +112,12 @@ impl Synchroniser {
         let is_initialised = remote_sync_state.initial_remote_data_synced()?;
         // Initialisation request was sent and successfully processed
         let is_sync_queue_initialised = remote_sync_state.sync_queue_initalised()?;
+        // Get site id from sever
+        if !is_initialised {
```

Comment:
Would potentially need to remove this, I think Roxy added this to server/lib.rs, made a note in this issue: https://github.com/openmsupply/open-msupply/issues/156